show/main.py

```diff
@@ -737,8 +744,35 @@ def mac(vlan, port, verbose):
     if port is not None:
         cmd += " -p {}".format(port)
 
+    if address is not None:
+        cmd += " -a {}".format(address)
+
+    if type is not None:
+        cmd += " -t {}".format(type)
+
+    if count:
+        cmd += " -c"
+
     run_command(cmd, display_cmd=verbose)
 
+@mac.command('aging-time')
+@click.pass_context
+def aging_time(ctx):
+    app_db = SonicV2Connector()
+    app_db.connect(app_db.APPL_DB)
+    table = "SWITCH_TABLE*"
+    keys = app_db.keys(app_db.APPL_DB, table)
+
+    if not keys:
+        click.echo("Aging time not configured for the switch")
+        return
+
+    for key in keys:
+        fdb_aging_time = app_db.get(app_db.APPL_DB, key, 'fdb_aging_time')
+        if fdb_aging_time is not None:
+            click.echo("Aging time for {} is {} seconds".format(key.split(':')[-1], fdb_aging_time))
+        else:
+            click.echo("Aging time not configured for the {}".format(key.split(':')[-1]))
```

Comment:
Do you have test case for this output?

Reply:
Yes. Please check test_show_mac_no_aging_time

Reply:
I think test_show_mac_no_aging_time is used to verify https://github.com/Azure/sonic-utilities/blob/a4776ab1e55e39892a006ede7e131c4049921cf2/show/main.py#L766-L768
Can you confirm?

Reply:
No. Its the other way around. test_show_mac_no_aging_time covers the absence of field value pair. I didn't add a test for absence of switch table(which is a more generic table and used by multiple applications).